custom_components/dmx/light.py

```diff
@@ -323,18 +323,21 @@ def dmx_values(self):
             return color_rgb_to_rgbw(*scaled_rgb)
         elif self._type == CONF_LIGHT_TYPE_DRGB:
             drgb = [self._brightness]
-            drgb.extend(self._rgb)
+            drgb.extend(self._rgb if self._brightness > 0 else [0, 0, 0])
+            _LOGGER.debug("drgb: " + ', '.join([str(x) for x in drgb]));
```

Comment:
Hey Ruth, sorry for the delay in getting to this. Can you help me understand why the RGB values should be zeroed for these lights? In my mind that's what the dimmer channel is there for.

Reply:
This was probably me being paranoid in the DRGB case then; however it won't do any harm for the RGB values to be zero if the dimmer channel is also zero, as the end result will be no light output either way.